src/components/split/splitInputs.js

```diff
@@ -0,0 +1,14 @@
+import Calculator from '../calculation/calculator.js';
+
+export default function splitInputs(inputs) {
+    let splitedInputs = [];
+    if (inputs.includes(':') === true) {
+        splitedInputs = inputs.split(':');
+    } else if (inputs.includes(',') === true) {
+        splitedInputs = inputs.split(',');
+    }
```

Comment:
let 대신 const와 push를 이용해봐도 좋을 것 같아요! 👍
```suggestion
    const splitedInputs = [];
    if (inputs.includes(':') === true) {
        splitedInputs.push(...inputs.split(':'))
    } else if (inputs.includes(',') === true) {
        splitedInputs.push(...inputs.split(','))
    }
```